### ISSUE TITLE
Screen reader is not announcing the group names 'Recently added samples' and 'Recently updated samples' while entering or existing from the related group of buttons.

### ISSUE THREAD

**Issue by Nitin-100** [microsoft/react-native-gallery#669]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
AT Used: Narrator

Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to 'Home' button and invoke it.
Navigate to the buttons present under "Recently added samples" and "Recently updated samples" groups
Now listen to the screen reader announcement.
Actual Result:
Screen Reader is not announcing the group names 'Recently added samples' and 'Recently updated samples' while entering or existing from the related group of buttons.
 
Expected Result:
Screen reader should announce the related group name when user enters or exists from the group of buttons.
 
User Impact:
User with visual impairments who rely on screen reader will be impacted and will face difficulties if group name will not be associated to the buttons and they might get confused with that behavior.

MAS Reference Link: [MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

Trap ID: [1.6 - Poor Grouping](https://aka.ms/PoorGrouping)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
